src/main/resources/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder/config.jelly

```diff
@@ -13,9 +13,13 @@
   <f:entry title="Application" field="appArtifact" description="[Required] Pattern to find newly built application.">
     <f:textbox />
   </f:entry>
+  
+  <f:entry title="Run Name" field="runName" description="[Optional] Name of your AWS Device Farm Run. Defaults to Environment variable BUILD_TAG.">
+    <f:textbox default="$${BUILD_TAG}" />
```

Comment:
Guessing the second "$" is escaping? Will it actually evaluate that variable without it?